src/Lokad.ILPack.csproj

```diff
@@ -23,6 +23,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <None Remove="Extensions.cs~RF6a0154.TMP" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
+    <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
```

Comment:
Could these be moved to an `ItemGroup` section that is specific to `netstandard2.0` and `net46` ?

Reply:
That's nice. Done.